modules/functional-test/src/main/java/com/intuit/wasabi/tests/library/util/Constants.java

```diff
@@ -106,4 +106,26 @@
 
     public static final String ROLE_LIST = "roleList";
 
+    /**
+     * This enum class holds the timeZones of various regions..added a few timezones
+     * feel free to add your timeZone if it is not there...
+     */
+    public static  enum TimeZoneEnum {
+        USA_PACIFIC("America/Los_Angeles"),
+        USA_MOUNTAIN("America/Denver"),
+        USA_CENTRAL("America/Chicago"),
+        USA_EAST("America/Detroit"),
+        USA_ALASKA("America/Anchorage") ;
+
+        private String timeZone;
```

Comment:
@sudheerkethireddy Instead of string timezone USA_ALASKA("America/Anchorage") , can you store TimeZone object itself; this will save creation of multiple TimeZone objects... e.g.g USA_ALASKA(new TimeZone("America/Anchorage"));